pkg/reconciler/integration/sink/controller.go

```diff
@@ -50,10 +53,27 @@ func NewController(
 	secretInformer := secretinformer.Get(ctx, "app.kubernetes.io/name")
 	eventPolicyInformer := eventpolicy.Get(ctx)
 	deploymentInformer := deploymentinformer.Get(ctx)
+	serviceInformer := service.Get(ctx)
 
-	cmCertificateInformer := cmcertinformer.Get(ctx)
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error getting cluster config: %v\n", err)
+		os.Exit(1)
+	}
 
-	serviceInformer := service.Get(ctx)
+	certManagerClient := certmanagerclient.NewForConfigOrDie(config)
+
+	resyncPeriod := 30 * time.Second
```

Comment:
there is a knative/pkg constant we could re-use `controller.DefaultResyncPeriod`, `import "knative.dev/pkg/controller"`